### DISCUSSION TITLE
Структура приложения ⚡️ Angular с примерами кода

### DISCUSSION THREAD

**Discussion by giscus** [xsltdev/angdev.ru#52] [Announcements]
# Структура приложения ⚡️ Angular с примерами кода

Сам фреймворк состоит из нескольких библиотек (или модулей), каждая из которых содержит в себе определенный функционал, а каждый модуль состоит из совокупности классов и их свойств и методов

https://angdev.ru/archive/angular9/application-structure/

**Comment by BaranovskiyNE** [comment]
Подушню немного - "функционал" != "функциональность", в первом же абзаце.
